build/tools/build-ports.py

```diff
@@ -194,8 +194,8 @@ def prepare_env():
 
     osversion = sh_str("awk '/\#define __FreeBSD_version/ { print $3 }' ${JAIL_DESTDIR}/usr/include/sys/param.h")
     login_env = e(',UNAME_r=${FREEBSD_RELEASE_VERSION% *},UNAME_v=FreeBSD ${FREEBSD_RELEASE_VERSION},OSVERSION=${osversion}')
-    sh('sed -i "" -e "s/,UNAME_r.*:/:/ ; s/:\(setenv.*\):/:\\1${login_env}:/" ${JAIL_DESTDIR}/etc/login.conf')
-    sh('cap_mkdb ${JAIL_DESTDIR}/etc/login.conf');
+    sh('sed -i "" -e "s/,UNAME_r.*:/:/ ; s/:\(setenv.*\):/:\\1${login_env}:/" ${JAIL_DESTDIR}/etc/login.conf.template')
+    sh('cap_mkdb -f ${JAIL_DESTDIR}/etc/login.conf ${JAIL_DESTDIR}/etc/login.conf.template')
```

Comment:
This change does not look right ,we dont want to mess with the ports build jail.